specification/personal-demographics.yaml

```diff
@@ -1088,7 +1088,7 @@ paths:
         * `/address/0` if the address to be replaced or removed is the first item in the list. If it is the second item in the list the path is `/address/1`, and so on.
         * `/address/-` when adding a new address.
 
-        An address consists of 5 lines of unstructured text, postcode and PAF (Postcode Address File) key.
+        An address consists of 5 lines of unstructured text, postcode and address keys, which can include a PAF (Postcode Address File) and a UPRN (Unique Property Reference Number).
```

Comment:
It's also a bit weird that we flip which out of the acronym/descript are in brackets vs what is in address. If this was prose I think you'd normally use brackets the first time you introduce an acronym and then just the acronym from then on

I don't know what is standard for these docs though